lib/orocos/process.rb

```diff
@@ -162,13 +162,17 @@ def orogen; model end
         # The set of task contexts for this process. This is valid only after
         # the process is actually started
         attr_reader :tasks
+        # Hash mapping task names with their respective IORs. This is valid only after
+        # the process is actually started
+        attr_reader :task_ior_mappings
```

Comment:
There is no need to declare the attr_reader. Declaring them for all instance variables was a bad habit I corrected since ...